test/integration/spec/connect.js

```diff
@@ -420,6 +420,48 @@ describe('connect', function() {
     });
   });
 
+  (isRTCRtpSenderParamsSupported ? describe.only : describe.skip)('dscpTagging', () => {
+    // eslint-disable-next-line no-undefined
+    [true, false, undefined].forEach((dscpTagging) => {
+      context(`when dscpTagging is ${typeof dscpTagging === 'undefined' ? 'not set' : dscpTagging ? 'set to true' : 'set to false'}`, () => {
+        const connectOptions = {};
+        if (dscpTagging !== undefined) {
+          connectOptions.dscpTagging = dscpTagging;
+        }
+
+        let peerConnections;
+        let thisRoom;
+        let thoseRooms;
+
+        before(async () => {
+          [thisRoom, thoseRooms, peerConnections] = await setup(connectOptions, { tracks: [] }, 0);
+          // RTCRtpSender.setParameters() is an asynchronous operation,
+          // wait for a little while until the changes are applied.
+          await new Promise(resolve => setTimeout(resolve, 5000));
+        });
+
+        ['audio', 'video'].forEach(kind => {
```

Comment:
@makarandp0 I don't think we should check for video here. We should write a unit test to make sure `setParameters` is not called for video, and called for audio. You can refer to the unit test I wrote in the previous PR.